### PR TITLE
Report static pinning options to the log (#691)

### PR DESCRIPTION
**Pull request by udi-speedb**

### CODE REVIEW

File: plugin/speedb/pinning_policy/scoped_pinning_policy.cc [hunk 1]
```diff
@@ -16,8 +16,12 @@
 
 #include "plugin/speedb/pinning_policy/scoped_pinning_policy.h"
 
+#include <inttypes.h>
+
```
Comment (Yuval-Ariel) [inline plugin/speedb/pinning_policy/scoped_pinning_policy.cc:20]: remove extra space

File: HISTORY.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ Fix RepeatableThread to work properly with on thread start callback feature (htt
 
 ### Enhancements
 * Unit Testing: Expose the disallow_trivial_move flag in the MoveFilesToLevel testing utility (#677).
+* Static Pinning: Report pinning policy name and paramteres to the log (#691).
```
Comment (Yuval-Ariel) [inline HISTORY.md:12]: parameters*

**Review by Yuval-Ariel — APPROVED** [review]
LGTM

File: HISTORY.md (hunk 2 above)
Comment (udi-speedb) [reply HISTORY.md:12]: Done

File: plugin/speedb/pinning_policy/scoped_pinning_policy.cc (hunk 1 above)
Comment (udi-speedb) [reply plugin/speedb/pinning_policy/scoped_pinning_policy.cc:20]: Done

**Review by Yuval-Ariel — APPROVED** [review]

**Pull request merged**